src/nvim/version.c

```diff
@@ -69,6 +69,7 @@ static char *features[] = {
 
 // clang-format off
 static int included_patches[] = {
+  1641,
```

Comment:
Usually all the commits not backported yet (so between 1641 and 1574) should be added as comment, to prevent weird merge conflicts later.. but looking at the next 4 lines people were obviously slacking recently.